front/pages/auth/signup/success.vue

```diff
@@ -10,6 +10,14 @@
         Please check your inbox and click on the link in your email to get
         started!
       </p>
+
+      <v-btn
```

Comment:
how about component for this button since its repeat itself in two places

Reply:
created component